src/node/historical_queries.h

```diff
@@ -518,43 +487,14 @@ namespace ccf::historical
 
     void fetch_entries_range(ccf::SeqNo from, ccf::SeqNo to)
```

Comment:
This function was previously trying some last minute optimisations - hoping to track exactly what had been previously requested, and not re-request it. Its now a simpler pass-through to support explicit re-fetches when needed, which are now primarily driven in `tick`s. Note that this means periodic fetch attempts when there is a ledger hole, which would previously have only been triggered on an additional user call.